src/monodroid/jni/monovm-properties.hh

```diff
@@ -10,23 +10,30 @@ namespace xamarin::android::internal
 {
 	class MonoVMProperties final
 	{
-		constexpr static size_t PROPERTY_COUNT = 2;
+		constexpr static size_t PROPERTY_COUNT = 3;
 
 		constexpr static char RUNTIME_IDENTIFIER_KEY[] = "RUNTIME_IDENTIFIER";
 		constexpr static size_t RUNTIME_IDENTIFIER_INDEX = 0;
 
 		constexpr static char APP_CONTEXT_BASE_DIRECTORY_KEY[] = "APP_CONTEXT_BASE_DIRECTORY";
 		constexpr static size_t APP_CONTEXT_BASE_DIRECTORY_INDEX = 1;
 
+		constexpr static char LOCAL_DATE_TIME_OFFSET_KEY[] = "System.TimeZoneInfo.LocalDateTimeOffset";
+		constexpr static size_t LOCAL_DATE_TIME_OFFSET_INDEX = 2;
+
 		using property_array = const char*[PROPERTY_COUNT];
 
 	public:
-		explicit MonoVMProperties (jstring_wrapper& filesDir)
+		explicit MonoVMProperties (jstring_wrapper& filesDir, jint localDateTimeOffset)
 		{
 			static_assert (PROPERTY_COUNT == N_PROPERTY_KEYS);
 			static_assert (PROPERTY_COUNT == N_PROPERTY_VALUES);
 
 			_property_values[APP_CONTEXT_BASE_DIRECTORY_INDEX] = strdup (filesDir.get_cstr ());
+
+			char localDateTimeOffsetBuffer[32];
+			snprintf (localDateTimeOffsetBuffer, sizeof(localDateTimeOffsetBuffer), "%d", localDateTimeOffset);
```

Comment:
Use `static_local_string<32>` here instead of `snprintf` (see the `strings.hh` header). It supports appending integers to the string and it does it much faster than `snprintf` and friends.